src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -40,6 +40,7 @@ import AppInfoItem from './AppInfoItem'
 import DeployModeModal from './DeployModeModal'
 import { useLanguageContext } from '../../../providers/LanguageProvider'
 import { EXPLORER_BASE_URL } from '../../../utils/config'
+import { currencyType as defaultCurrencyType } from '../../../utils/i18n'
```

Comment:
This is a string array, where currency type expects a string. Lets update the i18n file to have a defaultCurrencyType of USD. On line 17 of `../../../utils/i18n` add:
```
export const defaultCurrencyType = 'USD'
export const [currencyType] = RNLocalize.getCurrencies() || [
  defaultCurrencyType,
]
```

You can then change this import to:
```
import { defaultCurrencyType } from '../../../utils/i18n'
```

Reply:
On second thought, this should actually already be defaulting to USD in the `accountSlice.ts` if `RNLocalize.getCurrencies()` returned undefined. 

In order to encounter this bug, what is being returned by `RNLocalize.getCurrencies()`. I think the actual fix will be to check if the currency returned by `RNLocalize.getCurrencies()` is one of the supported currencies and if not we need to default to USD. We can only support currencies that are listed on coingecko via its API `https://api.coingecko.com/api/v3/coins/helium?localization=false&tickers=false&community_data=false&developer_data=false`